### DISCUSSION TITLE
开源摘星计划-ncnn贡献专栏

### DISCUSSION THREAD

**Discussion by nihui** [Tencent/ncnn#3776] [Ideas]
相关链接 https://github.com/weopenprojects/WeOpen-Star/issues/27
相关链接 https://github.com/weopenprojects/WeOpen-Star/blob/main/README.md

## 社区日常任务

* 提交可复现 bug 的 Issue
* 帮助解决问题的有意义的 Comment

## 文档类

* 收集资料补充和完善 FAQ https://github.com/Tencent/ncnn/blob/master/docs/faq.md
* 翻译现有资料为中英双语
* 增加新平台的benchmark数据
* 增加新平台的编译安装教程

## 编程类

* 新算子的pnnx转换和实现
* 优化现有算子的性能
* 针对新平台的移植
* 维护CI，测试自动化
* bug修复和改善代码质量
* 新AI算法移植ncnn（可实现在自己的repo里）

## 其他类

* 开源知识分享官：提交对本项目的学习笔记或开源相关笔记（视频/文章皆可）并发布在任意社区/博客/视频平台
* 发表论文并有提到ncnn

**Comment by tpoisonooo** [comment]
* 这分保值么？
* 请问能换啥，可以选 DJI 无人机/macpro 么？

**Reply by nihui** [reply]
> * 这分保值么？
> 
>     * 请问能换啥，可以选 DJI 无人机/macpro 么？

https://github.com/weopenprojects/WeOpen-Star#-%E5%A5%96%E9%A1%B9%E4%B8%8E%E6%BF%80%E5%8A%B1%E8%AE%BE%E7%BD%AE

看这里

**Reply by tpoisonooo** [reply]
啊~
* 领域内取 top，需要卷赢其他几个 1~4k 小 repo:  angel/fate/jinaAI/OpenMLDB
  * c++ 比 python 项目发 PR 有门槛优势
  * ncnn contributor 多
* 取 top100/top10，货币虽然是超发，但 base 大

"这个项目我王多鱼投了！”
